src/tagmanager.cpp

```diff
@@ -30,6 +30,15 @@ TagManager::TagManager()
         Ewmh::get().updateCurrentDesktop();
         tag_set_flags_dirty();
     });
+
+    setDoc(
+        "The tags (or virtual desktops or workspaces). This contains "
```

Comment:
The second sentence is missing something right? "This contains has an entry"?

Reply:
I've removed `has`. I was writing this part from scratch because there was no direct doc on `TagManager` in the original man page.

Reply:
Thanks!